### PR TITLE
Minor whitespace/line endings consistency changes

### PR DESCRIPTION
**Pull request by XhmikosR**
Non-whitespace diff: https://github.com/StevenBlack/hosts/pull/2623/files?w=1

### CODE REVIEW

**Comment by StevenBlack** [comment]
Thank you for this @XhmikosR.

A question: how does this square with our [`.editorconfig`](https://github.com/StevenBlack/hosts/blob/master/.editorconfig)?

Also, there's way more than whitespace and consistency changes in this PR. What's the motivation for those changes? Any downsides?

**Comment by XhmikosR** [comment]
I actually applied these in compliance with the current .editorconfig settings. It seems that previously these files landed with inconsistent settings.

**Comment by XhmikosR** [comment]
> Also, there's way more than whitespace and consistency changes in this PR. What's the motivation for those changes? Any downsides?

Just moving things around nothing fancy. I can revert those changes if you prefer :)

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 version: 2
 updates:
-  - package-ecosystem: "pip"
+  - package-ecosystem: "github-actions"
```
Comment (funilrys) [inline .github/dependabot.yml:3]: Is there a reason behind the order change ?

File: .github/workflows/depsreview.yaml [hunk 2]
```diff
@@ -13,5 +13,5 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: 'Dependency Review'
+      - name: Dependency Review
```
Comment (funilrys) [inline .github/workflows/depsreview.yaml:16]: If we start like this, then we should remove all quotes that are unnecessary 😉

Comment (XhmikosR) [reply .github/workflows/depsreview.yaml:16]: Well, it's without quotes already in most cases hence the change

File: .github/dependabot.yml (hunk 1 above)
Comment (XhmikosR) [reply .github/dependabot.yml:3]: https://github.com/StevenBlack/hosts/pull/2623#issuecomment-2034699665

I just keep actions first and then any package managers

File: .github/workflows/depsreview.yaml (hunk 2 above)
Comment (funilrys) [reply .github/workflows/depsreview.yaml:16]: I mean in all other yaml files 😉

**Comment by StevenBlack** [comment]
Hi @XhmikosR and @funilrys!

I'm happy 😄 and I'm merging this now.

The whole xz thing has me on-edge.  Not that @XhmikosR would be a vector 🤣 — I've decided to be much more sus generally, starting now, and in the future.

**Pull request merged**